33-room-occupancy-monitor/README.md

```diff
@@ -1,84 +1,82 @@
+
+Copy `main.py` over to the board with this command:
 
 ```
-pir: 1 motion events sensed
+python pyboard.py -d <serial port> --no-soft-reset -f cp main.py :/
 ```
 
-At startup, the PIR sensor may detect motion even if there is none. This is a one-off false positive that can be ignored, and the sensor should work as intended thereafter. If you encounter additional false positives, you may want to try increasing the sensor's threshold value, which makes the sensor less sensitive. This can be done by defining the macro `PIR_THRESHOLD_DEFAULT` at build-time (e.g. via `-DCMAKE_C_FLAGS="-DPIR_THRESHOLD_DEFAULT=50"`). If this macro isn't defined by the user, it'll default to 100 in pir.c:
+ Make sure to replace `<serial port>` with your serial port. `main.py` will start running after boot up.
 
-```c
-#ifndef PIR_THRESHOLD_DEFAULT
-#define PIR_THRESHOLD_DEFAULT 100
-#endif
-```
+### Hardware
 
-The value must be between 0 and 255. If you find that the sensor is not sensitive enough, you can try decreasing the threshold value.
+There are two primary pieces of hardware: the magnetic door switch and a PIR motion sensor.
+
+The magnetic door switch consists of two plastic terminals, one of which has a pair of wires coming out of it. When the terminals are brought into contact (or close proximity), an internal [reed switch](https://en.wikipedia.org/wiki/Reed_switch) is closed and the two wires are electrically connected. As shown in the SparkFun product link above, the terminal with the wires is typically mounted to a door frame and the other terminal is mounted to the door such that when the door is closed, the two terminals are adjacent, closing the switch. Since we're just testing things out, don't mount the terminals yet. Instead, keep them on your work surface and plug the wires into the GP21 and GND ports of the Pico (it doesn't matter which wire goes into which port). When the switch is closed, GP21 will connect to GND. Otherwise, GP21 will be driven high by an internal pull-up.
+
+The PIR motion sensor contains circuitry to convert the readings from the sensor into a simple high value on the data line for 2 seconds when motion is detected. Using the jumper cables, connect Pin 1 of the sensor to GND on the Pico, Pin 2 of the sensor to GP22 on the Pico, and Pin 2 of the sensor to 3V3(OUT) on the Pico.  GP22 will be driven low by an internal pull-down, to be brought high when movemnent is detected.
+
+The Notecarrier can be connected with a Qwiic cable with the following connctions:.
+
+  Qwiic Black (GND) to Pico GND
+  Qwiic Blue (SDA) to  Pico GP4 (Pin 6)
+  Qwiic Yellow (SCL) to Pico GP5 (Pin 7)
+
+Do not connect the Red cable of the Qwiic connector.  If you wish to power the Pico from the Notecarrier (which makes sense after testing) connect VMAIN on the Notecarrier to VSYS (Pin 39) on the Pico.
```

Comment:
You definitely need one or more photos of the wiring you discuss in this section. I’ll leave it up to you for where you think they logically fit in.

Reply:
I think I am going to rewrite it similar to how I did NF02, using a breadboard, a diagram from Fritzing, and a photo of the finished setup.  You are right that it's unclear here.